### DISCUSSION TITLE
Remove additional login when using `pac tool prt` or `pac tool cmt`

### DISCUSSION THREAD

**Discussion by filcole** [microsoft/powerplatform-build-tools#934] [Ideas]
`pac tool prt` or `pac tool cmt` start the classic windows only Plugin Registration Tool or Configuration Migraiton Tool.  They always start with a login screen.

Please consider updating the tools to pass currently authenticated environment (`pac env who`), so that these tools can start without the user having to login again.

Doing so: 
* is a time saver for users of prt and cmt.
* increases security, due to less logins required
* encourages people to use the standard PRT instead of the forked version in XrmToolbox.

Note: This was raised in discord by me last year, and also raised by someone else in the Power Platform Pro-Code call on 21/06/2024.

**Comment by jondoesflow** [comment]
This is a great idea !!

**Comment by petrochuk** [comment]
Unfortunately PRT and CMT use incompatible auth framework ADAL (Active Directory Authentication Library) while PAC CLI uses more modern [Microsoft Authentication Library (MSAL)](https://learn.microsoft.com/en-us/entra/identity-platform/msal-overview])
